openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -180,14 +180,17 @@ def get_invalid(cls, instance):
                         redshift_AOV_prefix
                     ))
                     invalid = True
-                # get aov format
-                aov_ext = cmds.getAttr(
-                    "{}.fileFormat".format(aov), asString=True)
 
-                default_ext = cmds.getAttr(
-                    "redshiftOptions.imageFormat", asString=True)
```

Comment:
I've removed the `asString` argument since the returned values were returned as integers anyway.